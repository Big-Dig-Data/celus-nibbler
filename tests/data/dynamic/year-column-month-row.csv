Metric,Year,Jan,Feb,Mar,Apr,May,Jun,Jul,Aug,Sep,Oct,Nov,Dec
M1,2020,1,2,3,4,5,6,7,8,9,10,11,12
M2,2020,1,3,5,7,9,11,2,4,6,8,10,12
M1,2021,2,3,4,5,6,7,8,9,10,11,12,1
M2,2021,3,4,5,6,7,8,9,10,11,12,1,2
