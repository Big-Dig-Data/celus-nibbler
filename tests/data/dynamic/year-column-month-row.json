{
  "kind": "non_counter.generic",
  "version": 1,
  "parser_name": "year-column-month-row",
  "data_format": {
    "name": "simple_format"
  },
  "platforms": [
    "Platform1",
    "Platform2"
  ],
  "metrics_to_skip": [],
  "titles_to_skip": [],
  "dimensions_to_skip": {},
  "metric_aliases": [],
  "dimension_aliases": [],
  "heuristics": {
    "conds": [
      {
        "pattern": "Metric",
        "coord": {
          "row": 0,
          "col": 0
        },
        "kind": "regex"
      },
      {
        "pattern": "Year",
        "coord": {
          "row": 0,
          "col": 1
        },
        "kind": "regex"
      }
    ],
    "kind": "and"
  },
  "areas": [
    {
      "data_headers": {
        "roles": [
            {
                "source": {
                    "coord": {
                      "row": 0,
                      "col": 2
                    },
                    "direction": "right"
                },
                "role": "date"
            }
        ],
        "data_direction": "down",
        "data_cells": {
          "coord": {
            "row": 1,
            "col": 2
          },
          "direction": "right"
        },
        "data_cells_options": {
            "use_header_year": false
        }
      },
      "dates": {
        "source": {
          "coord": {
            "row": 1,
            "col": 1
          },
          "direction": "down"
        }
      },
      "titles": null,
      "metrics": {
        "source": {
          "coord": {
            "row": 1,
            "col": 0
          },
          "direction": "down"
        }
      },
      "title_ids": [],
      "dimensions": [],
      "kind": "non_counter.generic"
    }
  ]
}
