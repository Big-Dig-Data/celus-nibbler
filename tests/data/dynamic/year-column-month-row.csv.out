"2020-01-01","2020-01-31","","","","","","","","","M1","1"
"2020-02-01","2020-02-29","","","","","","","","","M1","2"
"2020-03-01","2020-03-31","","","","","","","","","M1","3"
"2020-04-01","2020-04-30","","","","","","","","","M1","4"
"2020-05-01","2020-05-31","","","","","","","","","M1","5"
"2020-06-01","2020-06-30","","","","","","","","","M1","6"
"2020-07-01","2020-07-31","","","","","","","","","M1","7"
"2020-08-01","2020-08-31","","","","","","","","","M1","8"
"2020-09-01","2020-09-30","","","","","","","","","M1","9"
"2020-10-01","2020-10-31","","","","","","","","","M1","10"
"2020-11-01","2020-11-30","","","","","","","","","M1","11"
"2020-12-01","2020-12-31","","","","","","","","","M1","12"
"2020-01-01","2020-01-31","","","","","","","","","M2","1"
"2020-02-01","2020-02-29","","","","","","","","","M2","3"
"2020-03-01","2020-03-31","","","","","","","","","M2","5"
"2020-04-01","2020-04-30","","","","","","","","","M2","7"
"2020-05-01","2020-05-31","","","","","","","","","M2","9"
"2020-06-01","2020-06-30","","","","","","","","","M2","11"
"2020-07-01","2020-07-31","","","","","","","","","M2","2"
"2020-08-01","2020-08-31","","","","","","","","","M2","4"
"2020-09-01","2020-09-30","","","","","","","","","M2","6"
"2020-10-01","2020-10-31","","","","","","","","","M2","8"
"2020-11-01","2020-11-30","","","","","","","","","M2","10"
"2020-12-01","2020-12-31","","","","","","","","","M2","12"
"2021-01-01","2021-01-31","","","","","","","","","M1","2"
"2021-02-01","2021-02-28","","","","","","","","","M1","3"
"2021-03-01","2021-03-31","","","","","","","","","M1","4"
"2021-04-01","2021-04-30","","","","","","","","","M1","5"
"2021-05-01","2021-05-31","","","","","","","","","M1","6"
"2021-06-01","2021-06-30","","","","","","","","","M1","7"
"2021-07-01","2021-07-31","","","","","","","","","M1","8"
"2021-08-01","2021-08-31","","","","","","","","","M1","9"
"2021-09-01","2021-09-30","","","","","","","","","M1","10"
"2021-10-01","2021-10-31","","","","","","","","","M1","11"
"2021-11-01","2021-11-30","","","","","","","","","M1","12"
"2021-12-01","2021-12-31","","","","","","","","","M1","1"
"2021-01-01","2021-01-31","","","","","","","","","M2","3"
"2021-02-01","2021-02-28","","","","","","","","","M2","4"
"2021-03-01","2021-03-31","","","","","","","","","M2","5"
"2021-04-01","2021-04-30","","","","","","","","","M2","6"
"2021-05-01","2021-05-31","","","","","","","","","M2","7"
"2021-06-01","2021-06-30","","","","","","","","","M2","8"
"2021-07-01","2021-07-31","","","","","","","","","M2","9"
"2021-08-01","2021-08-31","","","","","","","","","M2","10"
"2021-09-01","2021-09-30","","","","","","","","","M2","11"
"2021-10-01","2021-10-31","","","","","","","","","M2","12"
"2021-11-01","2021-11-30","","","","","","","","","M2","1"
"2021-12-01","2021-12-31","","","","","","","","","M2","2"
